lib/slack/real_time/concurrency/celluloid.rb

```diff
@@ -73,7 +73,14 @@ def start_async(client)
           end
 
           def run_client_loop
+            current_actor.every @client.websocket_ping do
```

Comment:
I believe we want this wrapped in a conditional `if @client.run_ping?`.

Reply:
I implemented this with tests in https://github.com/slack-ruby/slack-ruby-client/pull/236/commits/019594953c372736172ce402d83b482888fa8921